setup.py

```diff
@@ -31,15 +31,14 @@
     long_description=README,
     long_description_content_type='text/markdown',
     license="3-clause BSD",
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*',
+    python_requires='>2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*',
```

Comment:
In CMake (https://github.com/lbl-srg/funnel/pull/63/files#diff-cdd48abbd3eb8d1c54077449fc74a8de1f29805d2be5d8e5232b7aab76ea7a6fL16) and in `requirements.txt` Python 2.7 is no longer supported. Why is `>2.7` here if the 2.x series is not tested?